samples/Erlang/release.es

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env escript
```

Comment:
I probably should have made that clear before, but I'm not from GitHub staff, I can only give an opinion ;)